components/menu/index.css

```diff
@@ -396,7 +400,7 @@
 	padding-block-end: var(--mod-menu-item-bottom-edge-to-text, var(--spectrum-menu-item-bottom-edge-to-text));
 	padding-inline: var(--mod-menu-item-label-inline-edge-to-content, var(--spectrum-menu-item-label-inline-edge-to-content));
 
-	margin: calc((var(--spectrum-menu-item-focus-indicator-offset) + var(--spectrum-menu-item-focus-indicator-border-width)) * var(--spectrum-menu-item-spacing-multiplier));
+	margin: calc((var(--spectrum-menu-item-focus-indicator-offset) + var(--mod-menu-item-focus-indicator-width, var(--spectrum-menu-item-focus-indicator-width))) * var(--spectrum-menu-item-spacing-multiplier));
```

Comment:
This would be a good use-case for moving the --mod definition to the first declaration of --spectrum-menu-item-focus-indicator-width since it's being used twice. See line 90 above.

```
--spectrum-menu-item-focus-indicator-width: var(--mod-menu-item-focus-indicator-width, var(--spectrum-border-width-200));
```